### ISSUE TITLE
Filter tag doesn't work with Choices tag

### ISSUE THREAD

**Issue by bulutaygunes** [heartexlabs/label-studio#2967]
**Describe the bug**
"Filter" tag doesn't filter the choices under the "Choices" tag.

**To Reproduce**
1. Create a project.
2. Go to Settings --> Labeling Interface.
3. Use the following labeling configuration as an example:
```xml
<View>
    <View style="display: flex; align-items: start; gap: 8px; flex-direction: column; white-space: pre-wrap;">
        <Text name="text" value="$text" granularity="word"/>
    </View>
    <Filter name="filter" toName="category" hotkey="shift+f" minlength="0" placeholder="Search"/>
    <Choices name="category" toName="text" choice="multiple" showInLine="false" required="true">
        <Choice value="cat"/>
        <Choice value="dog"/>
        <Choice value="bird"/>
        <Choice value="frog"/>
        <Choice value="bear"/>
        <Choice value="bug"/>
    </Choices>
</View>
```
4. Try to type something in the filter text box in UI Preview section on the right.
5. Observe that nothing happens as you type; it doesn't filter out any of the choices.

**Expected behavior**
I would expect to see a limited set of choices when I start to type something in the filter text box.

**Screenshots**
We should have seen "bird", "bear" and "bug" here since we typed "b" in the search box:
![Screenshot from 2022-09-20 21-43-27](https://user-images.githubusercontent.com/53745901/191339026-720051e9-0934-4019-93f4-34e4be4b5900.png)

**Environment:**
 - OS: Fedora Linux 35
 - Label Studio Version: 1.5.0post0

**Comment by GiacomoCherry** [comment]
Hi, same issue here but I've noticed that it works properly if you try it on the [online playground](https://labelstud.io/playground/).

**Comment by peter-doggart** [comment]
I'm also seeing this issue on my deployed version v1.6.0.

**Comment by averdones** [comment]
Is there a time estimate for this fix? I've been having a look at it, but I'm still figuring out where to look in the code.

I checked that the bug was introduced in version 1.5.0. In version 1.4.1, it's working for me.

**Comment by pauliusgerve** [comment]
seems like it is still unresolved as of 1.7.0

**Comment by hlomzik** [comment]
How do you check it? Are you that length of entered criteria is bigger than `minlength` which is 3?
Just checked it and filtering works fine. Could you provide us with your config and filtering criteria you use?

**Comment by hlomzik** [comment]
it was [fixed in October](https://github.com/heartexlabs/label-studio-frontend/pull/918), so I'm closing this issue

**Comment by averdones** [comment]
Sorry for the delay. Indeed, it is fixed for me now in version 1.7. Thank you for the work!
